starboard/android/shared/media_codec_bridge.cc

```diff
@@ -177,6 +178,11 @@ scoped_ptr<MediaCodecBridge> MediaCodecBridge::CreateAudioMediaCodecBridge(
     return scoped_ptr<MediaCodecBridge>(NULL);
   }
 
+  if (MediaCodecBridgeEradicator::GetInstance()->IsEnabled()) {
```

Comment:
The check of `IsEnabled()` can be moved into `CheckEradicatingAndMaybeWait()`.

Reply:
The code has been cleanly separated by IsEnabled(), allowing for easy removal if the launch does not go as planned.